pkg/apiserver/rest/dualwriter_mode2.go

```diff
@@ -248,6 +268,36 @@ func (d *DualWriterMode2) Update(ctx context.Context, name string, objInfo rest.
 	return d.Storage.Update(ctx, name, objInfo, createValidation, updateValidation, forceAllowCreate, options)
 }
 
+func (d *DualWriterMode2) Destroy() {
+	d.Storage.Destroy()
+	d.Legacy.Destroy()
+}
+
+func (d *DualWriterMode2) GetSingularName() string {
+	return d.Legacy.GetSingularName()
+}
+
+func (d *DualWriterMode2) NamespaceScoped() bool {
+	return d.Legacy.NamespaceScoped()
+}
+
+func (d *DualWriterMode2) New() runtime.Object {
+	return d.Legacy.New()
+}
+
+func (d *DualWriterMode2) NewList() runtime.Object {
+	return d.Storage.NewList()
+}
+
+func (d *DualWriterMode2) Watch(ctx context.Context, options *metainternalversion.ListOptions) (watch.Interface, error) {
+	klog.Error("Watch not implemented")
```

Comment:
If I remove it, the compiler will trow an error, since the interface is not fully satisfied ([StandardStorage has a watcher interface](https://pkg.go.dev/k8s.io/apiserver@v0.29.2/pkg/registry/rest#StandardStorage))

Reply:
we could make our own temporary copy of that interface and remove `Watch`. thoughts?

Reply:
yep, I def can do that :) I wanted to understand what would be the concern of including watch? Do we have already any consumers in place which will attempt to call this?

Reply:
i'm mostly thinking of any k8s tooling that relies on API discovery to confirm that the resources have support for the verbs that the tool needs. this specific example doesn't include watch, but here's how velero finds supported resources: https://github.com/vmware-tanzu/velero/blob/4d48273a24969b748b6c2c68f376928053188102/pkg/discovery/helper.go#L265-L267

it's probably not a huge concern at this point